typescript/packages/well-log-viewer/src/WellLogViewer.stories.tsx

```diff
@@ -291,6 +291,56 @@ export const Default: StoryObj<typeof StoryTemplate> = {
     render: (args) => <StoryTemplate {...args} />,
 };
 
+export const ColorFunc: StoryObj<typeof StoryTemplate> = {
```

Comment:
ATM this is not really a color function example, so maybe it can be named as such when it becomes a real example (if I understand the intention correctly)

Reply:
The color function doesn't work so I commented it. I could add TBD (to be done) to the story name so that for now storybook users will see this example doesn't work yet